src/helpers/graph-util.js

```diff
@@ -128,11 +128,16 @@ const GraphUtil = {
     graph.createLine("grid", minYAxisGrid, maxYAxisGrid, "black");
 
     if (showBoundingLabels) {
-      graph.createLabel("label", minXAxisGrid, minGridX.toString(), "left");
-      graph.createTick("tick", minXAxisGrid, "x");
-
-      graph.createLabel("label", maxXAxisGrid, maxGridX.toString(), "right");
-      graph.createTick("tick", maxXAxisGrid, "x");
+      
+      if(minGridY > -4) {
```

Comment:
Where did this 4 heuristic come from, was it just trial and error? Might be nice to have a comment here explaining where it's from

Reply:
done.